### ISSUE TITLE
WooCommerce mini cart refreshing and how to disable

### ISSUE THREAD

**Issue by cdsaenz** [bootscore/bootscore#154]
Been working a lot with your fabulous theme, with the bootcommerce-child-main variant specifically. In the meantime I decided to roll my own multilingual library to apply to the theme. It's been a lot of work and mostly it does work (url rewriting etc)! But well it involves refreshing, transients etc. The one issue I'm having now is that the mini-cart offcanvas won't update its content to the locale I did set before. I turned off the wc template cache, etc but only a full reload works. Also sometimes it won't update the products or quantities. If I do load the mini-cart.php itself via php it does update.. but the JS won't load apparently.

A couple of questions in case you can provide me some idea/guidance:
- I see per your code that calling get_refreshed_fragments (via ajax I think) might be the solution. But I couldn't figure out how to call it as the page is loaded to refresh just the mini cart.
- More simple: how do I disable completely the mini cart? I removed the code from the header (just kept the cart info part) but I get some JS errors, so maybe I'm doing something wrong or missing some configuration. 

Thanks a lot as usual for the hard work!!

**Comment by crftwrk** [comment]
Hi,

1. The Ajax Mini Offcanvas Cart is a pluggable function in `woocommerce/woocommerce-functions.php` in line 29. You can disable it by storing the empty function your child:

```php
function register_ajax_cart() {
}
add_action('after_setup_theme', 'register_ajax_cart');
```

2. The Ajax Cart button is a pluggable function as well in `woocommerce/woocommerce-functions.php` in line 58.

3. Remove all cart items as you did in header.php

4. Override bootScore's mini-cart.php by storing the default woocommerce file https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/cart/mini-cart.php in your child `woocommerce/cart/mini-cart.php`

Usually the ajax cart should now be completely disabled. If you still get an JS error, try to remove snippet in main-theme `woocommerce/js/woocommerce.js` lines 2 to 14.

Let me know if this works fine or not, so we can maybe move the cart JS to a pluggable functions too.

**Comment by cdsaenz** [comment]
Great, now it's working fine, the `woocommerce.js` bit was what I needed to disable (I dequeued the main theme version and copied it to the child, commented out just in case). I also kept your mini-cart.php. The info button stays! That works perfectly. 

I am hopeful to fix the problem and get the mini cart back, it's fab. The problem at the moment seems identified. When I switch to another language, the localized strings in the mini cart form won't update with the new locale: `woocommerce_mini_cart() `seems to be the culprit as it loads via JS. I need to force the refresh on that bit somehow on page load or rewrite the buttons/totals logic.  

Fun continues! Thanks!

![image](https://user-images.githubusercontent.com/2335805/168616805-4f6fd921-346b-4f5e-836c-dadf1d704d20.png)
